internal/gengapic/testdata/empty_example.want

```diff
@@ -4,6 +4,7 @@ func ExampleNewClient() {
 	if err != nil {
 		// TODO: Handle error.
 	}
+	defer c.Close()
```

Comment:
Nit, feel free to ignore: line break to separate this from the `// TODO`

Reply:
Done.